src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -30,13 +31,13 @@ A headless browser is a great tool for automated testing and server environments
 don't need a visible UI shell. For example, you may want to run some tests against
 a real web page, create a PDF of it, or just inspect how the browser renders an URL.
 
-Note: Headless mode is available on Mac and Linux in Chrome 59. Windows support
-is coming soon!
+Note: Headless mode is available on Mac and Linux in **Chrome 59**.
```

Comment:
Then the way to do it is to call this out right at the top. Instead of: 

Headless Chrome is a way to run the Chrome browser in a headless environment.

Starting in Chrome 59, headless Chrome ...